config/dataset/test_ci_segmentation_binary.yaml

```diff
@@ -2,7 +2,7 @@
 dataset:
   # dataset-wide
   name:
-  input_dim: 256
+  input_dim: 32
```

Comment:
32?

Reply:
The test data is much smaller than previous data (current is around 100x100 pixels, previous was around 1000x1000). To test the training pipeline the input dimension was reduced to 32. If you prefer, we could also maintain 256 (or 512, which is our standard for complete trainings), and simply overwrite this parameter during testing in the CI pipeline.